### PR TITLE
Chore: Signed payload support for aws

### PR DESCRIPTION
**Pull request by godruoyi**
This PR is to add support for Signed Payload when signing with AWS. this is my first rust PR, lmk  if I'm doing something wrong.

close https://github.com/Xuanwo/reqsign/issues/10

### CODE REVIEW

**Comment by godruoyi** [comment]
@Xuanwo PTAL

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 /target
 Cargo.lock
 .env
+.idea
```
Comment (Xuanwo) [inline .gitignore:4]: Please add this in your git global ignore instead, thanks!

**Review by Xuanwo — COMMENTED** [review]
Wow, mostly LGTM!

File: .gitignore (hunk 1 above)
Comment (godruoyi) [reply .gitignore]: thank you @Xuanwo, updated it, please take a look again.

**Pull request merged**